packages/babel-compat-data/data/plugin-bugfixes.json

```diff
@@ -0,0 +1,125 @@
+{
+  "transform-async-to-generator": {
+    "chrome": "55",
+    "edge": "15",
+    "firefox": "52",
+    "safari": "10.1",
+    "node": "7.6",
+    "ios": "10.3",
+    "samsung": "6",
+    "opera": "42",
+    "electron": "1.6"
+  },
+  "bugfix/transform-async-arrows-in-class": {
+    "chrome": "55",
+    "edge": "15",
+    "firefox": "52",
+    "safari": "11",
```

Comment:
~Safari should be 10.1 / 10.3 here I think? Same as async-to-generator. I may have had references to Safari 11 in the plugin since that's what I had chosen to target, but the fix should work in all ES-Modules-supporting versions of Safari (10.1+ desktop / 10.3+ ios)~

Edit: I keep thinking this list is "versions that are fixed by X", but it's actually "oldest version where this plugin is not needed", right?

Reply:
> oldest version where this plugin is not needed

:+1: